DataManagementSystem/Service/DataIntegrityHandler.py

```diff
@@ -9,9 +9,9 @@
 
 """
 
-## imports
-from types import DictType, IntType, LongType, ListType, StringTypes
-## from DIRAC
+# imports
+from types import IntType, LongType, StringTypes
```

Comment:
why keeping those ?

Reply:
Because these are types that change in python 3, and this PR don't want to mess with those.